Dockerfile

```
@@ -1,66 +1,37 @@
-FROM golang:1.11 as builder
+FROM golang:1.13.1 as builder
```

```suggestion
FROM golang:1.13 as builder
```
so we always pick the security patches automatically